mcs/class/System.Transactions/System.Transactions/Transaction.cs

```diff
@@ -218,6 +218,17 @@ public void Dispose ()
 			return new Enlistment ();
 		}
 
+		[MonoTODO ("Only Local Transaction Manager supported. Cannot have more than 1 durable resource per transaction.")]
+		[PermissionSetAttribute (SecurityAction.LinkDemand)]
+		public Enlistment PromoteAndEnlistDurable (
+			Guid manager,
+			IPromotableSinglePhaseNotification promotableNotification,
+			IEnlistmentNotification notification,
```

Comment:
this should be ISinglePhaseNotification: https://msdn.microsoft.com/en-us/library/system.transactions.transaction.promoteandenlistdurable(v=vs.110).aspx